R/utils.R

```diff
@@ -120,29 +124,82 @@ expose_stan_fns <- function(files, target_dir, ...) {
 #'
 #' # Load the script used to generate these examples
 #' # Optionally source this script to regenerate the example
-#' readLines(enw_example(type = "script"))
-enw_example <- function(type = "nowcast") {
-  type <- match.arg(
-    type,
-    choices = c(
-      "nowcast", "preprocessed_observations", "observations", "script"
-    )
-  )
+#' readLines(enw_example(type = "scripts"))
+enw_example <- function(
+  type = c("nowcast", "preprocessed_observations", "observations", "scripts")
+) {
+  type <- match.arg(type)
```

Comment:
This reorg would ideally happen as part of #156 vs in random PRs

Reply:
fair, but it was litter in front of me while dealing with example updates.